src/components/ProfileEditEmailPage.js

```diff
@@ -86,11 +88,10 @@ const ProfileEditEmailPage = () => {
           fullWidth
           label="New email"
           name="email"
-          placeholder="unicorn@usepow.app"
           autoComplete="email"
           helperText={
             <>
-              You current email address is <strong>test@test.com</strong>.
+              Your current POW! email is <strong>{yourCurrentPOWemail}</strong>.
```

Comment:
Good catch!

Reply:
This could (should) be its own Pull Request, create an issue labeled bug. Create a branch, fix just this and submit Pull Request. And we merge.

Reply:
https://github.com/raae/pow-app/pull/131  = {currentEmail}